test/library/input.spec.js

```diff
@@ -524,5 +526,38 @@ describe('NumberFormat as input', () => {
       wrapper.setState({ value: '123.1234' });
       expect(wrapper.find('input').instance().value).toEqual('$123.1234');
     });
+
+    it('should show the right number of zeros in all cases', () => {
```

Comment:
Can you add the issue number in the testcase.

Reply:
May be as comment.

Reply:
@s-yadav Done!